sdk/batch/Microsoft.Azure.Batch/changelog.md

```diff
@@ -1,5 +1,19 @@
 # Microsoft.Azure.Batch release notes
 
+## Changes in 12.0.0
+### Features
+- Added ability to specify a collection of public IPs on `CloudPool` via the new `PublicIPs` property. This guarantees nodes in the Pool will have an 
+  IP from the list user provided IPs.
+- Added ability to mount remote file-systems on each node of a pool via the `MountConfiguration` property on `CloudPool`.
+- **[Breaking]** When not specified, the default value for `WaitForSuccess` on `StartTask` is now `true` (was `false`).
```

Comment:
Either add a note about SIG to 11 or here

Reply:
Fixed